readthedocs/core/resolver.py

```diff
@@ -63,23 +64,28 @@ def base_resolve_path(
             subproject_slug=None,
             subdomain=None,
             cname=None,
+            version_type=None
     ):
         """Resolve a with nothing smart, just filling in the blanks."""
         # Only support `/docs/project' URLs outside our normal environment. Normally
         # the path should always have a subdomain or CNAME domain
         # pylint: disable=unused-argument
-        if subdomain or cname or (self._use_subdomain()):
-            url = '/'
-        else:
-            url = '/docs/{project_slug}/'
+        if version_type != EXTERNAL:
+            if subdomain or cname or (self._use_subdomain()):
+                url = '/'
+            else:
+                url = '/docs/{project_slug}/'
 
-        if subproject_slug:
-            url += 'projects/{subproject_slug}/'
+            if subproject_slug:
+                url += 'projects/{subproject_slug}/'
 
-        if single_version:
-            url += '{filename}'
+            if single_version:
+                url += '{filename}'
+            else:
+                url += '{language}/{version_slug}/{filename}'
         else:
-            url += '{language}/{version_slug}/{filename}'
+            # we serve external version media files from media storage
+            url = '/media/external/html/{project_slug}/{version_slug}/{filename}'
```

Comment:
This is the part we need to logic around. We _still_ don't have a setting for the domain hosting the external builds. That's the entire point of this PR, to link properly to `external-builds.readthedocs.io` in prod and something similar in dev.

Reply:
are you planing to add the domain:host here?
something like this:
`settings.EXTERNAL_DOC_URL = 'external-builds.readthedocs.io'`
`{external_doc_url}/media/external/html/{project_slug}/{version_slug}/{filename}`